lib/posthog/client.rb

```diff
@@ -156,7 +160,7 @@ def queued_messages
     end
 
     # TODO: In future version, rename to `feature_flag_enabled?`
-    def is_feature_enabled( # rubocop:disable Naming/PredicateName
+    def is_feature_enabled(# rubocop:disable Naming/PredicateName
```

Comment:
weird?

```suggestion
    def is_feature_enabled( # rubocop:disable Naming/PredicateName
```

Reply:
yeah, i did autoformat in rubymine
silly computers fighting each other i guess